sqlflow_models/dnnclassifier.py

```diff
@@ -38,7 +38,7 @@ def call(self, inputs, training=True):
             x = hidden_layer(x)
         return self.prediction_layer(x)
 
-def optimizer(learning_rate=0.001):
+def optimizer(learning_rate=0.01):
```

Comment:
Why update the default value in a package upgrading PR?

Reply:
There have many changes since TF2.0.0b1. Now the default learning rate of Adagrad is too small to stably meet the loss requirement of the unit test. (In fact, even in TF2.0.0b1, the unit test randomly fails to meet the loss criteria)